### PR TITLE
Support for Drayton Wiser Smart TRV (WV704R0A0902)

### PR DESCRIPTION
**Pull request by arcanefoam**

### CODE REVIEW

**Comment by sjorge** [comment]
General comment as on the other PR's, maybe reference it as 'Drayton/Eberle Wiser', there may be others but at least both of these are the same, just the little logo on the rotating knob is different.

File: docs/devices/WV704R0A0902.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+## Comisioning
+After installing the TRV twist the cap in the **-** direction and hold for 
+2 seconds until the blue LED lights up. 
+
+## Device hard reset
+If the device fails to pair/join the network (`red:yellow:blue` on paring mode) or you changed the network id/channel, connect to another network, bought the TRV second hand, you can perform a factory reset to start fresh.
+
+1. Make sure that the TRV is NOT in pairing mode.
+2. Twist the cap in the **-** direction and hold till blue light turns off and then center light blinks red (about 10 seconds).
```
Comment (sjorge) [inline docs/devices/WV704R0A0902.md]: Manual says 15 sec...

```
Turn the cap in the minus direction for >15 s and only release
it when the red LED no longer flashes.
If successful, the valve is opened, the right and left LEDs
flash, the middle LED lights up orange for 5 minutes.
```

File: docs/devices/WV704R0A0902.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+    "value_template": "{{ value_json.battery }}"
+    "unique_id": "<DEVICE_ADDR>_1_battery"
+    "availability_topic": "zigbee2mqtt/bridge/state"
+    "json_attributes_topic": "zigbee2mqtt/<FRIENDLY_NAME>"
+    "state_topic": "zigbee2mqtt/<FRIENDLY_NAME>"
+    "unit_of_measurement": "%"
+    "device_class": "battery"
+    "device":
+      "identifiers": "zigbee2mqtt_<DEVICE_ADDR>"
+      "manufacturer": "Drayton Wiser"
```
Comment (sjorge) [inline docs/devices/WV704R0A0902.md]: does this need to be updated too?

File: docs/devices/WV704R0A0902.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+    "name": "<FRIENDLY_NAME>_boost"  
+    "value_template": "{{ value_json.boost }}"
+    "unique_id": "<DEVICE_ADDR>_65027_boost"
+    "availability_topic": "zigbee2mqtt/bridge/state"
+    "json_attributes_topic": "zigbee2mqtt/<FRIENDLY_NAME>"
+    "state_topic": "zigbee2mqtt/<FRIENDLY_NAME>"
+    "icon": "mdi:thermometer-alert"
+    "device":
+      "identifiers": "zigbee2mqtt_<DEVICE_ADDR>"
+      "manufacturer": "Drayton Wiser"
```
Comment (sjorge) [inline docs/devices/WV704R0A0902.md]: does this need to be updated too?

File: docs/devices/WV704R0A0902.md [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+    "name": "<FRIENDLY_NAME>_heating_demand"  
+    "value_template": "{{ value_json.pi_heating_demand }}"
+    "unique_id": "<DEVICE_ADDR>_513_piHeatingDemand"
+    "availability_topic": "zigbee2mqtt/bridge/state"
+    "json_attributes_topic": "zigbee2mqtt/<FRIENDLY_NAME>"
+    "state_topic": "zigbee2mqtt/<FRIENDLY_NAME>"
+    "icon": "mdi:fire"
+    "device":
+      "identifiers": "zigbee2mqtt_<DEVICE_ADDR>"
+      "manufacturer": "Drayton Wiser"
```
Comment (sjorge) [inline docs/devices/WV704R0A0902.md]: does this need to be updated too?

File: docs/devices/WV704R0A0902.md [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+    "value_template": "{{ value_json.linkquality }}"
+    "unique_id": "<DEVICE_ADDR>_2821_lastMessageLqi"
+    "availability_topic": "zigbee2mqtt/bridge/state"
+    "json_attributes_topic": "zigbee2mqtt/<FRIENDLY_NAME>"
+    "state_topic": "zigbee2mqtt/<FRIENDLY_NAME>"
+    "icon": "mdi:link-variant"
+    "unit_of_measurement": "-"
+    "device":
+      "identifiers": "zigbee2mqtt_<DEVICE_ADDR>"
+      "manufacturer": "Drayton Wiser"
```
Comment (sjorge) [inline docs/devices/WV704R0A0902.md]: does this need to be updated too?

File: docs/devices/WV704R0A0902.md [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
+*To contribute to this page, edit the following
+[file](https://github.com/Koenkk/zigbee2mqtt.io/blob/master/docs/devices/WV704R0A0902.md)*
+
+# Drayton Wiser Smart Radiator Thermostat (iTRV)
+
+| Model | WV704R0A0902  |
+| Vendor  | Schneider Electric (Drayton Wiser,  Eberle Wiser) |
+| Description | Smart Radiator Thermostat with twist-top boost |
+| Supports | temperature, battery, keypad lock, heating demand, boost |
+| Picture | ![Drayton Wiser iTRV](../images/devices/WV704R0A0902.jpg) |
```
Comment (sjorge) [inline docs/devices/WV704R0A0902.md]: Should also be 'Schneider Electric'

File: docs/devices/WV704R0A0902.md [hunk 7]
```diff
@@ -0,0 +1,211 @@
+---
+title: "Drayton Wiser iTRV control via MQTT"
+description: "Integrate your Drayton Wiser Smart Radiator Thermostat (iTRV) via Zigbee2mqtt with whatever smart home
+ infrastructure you are using without the vendors bridge or gateway."
+---
+
+*To contribute to this page, edit the following
+[file](https://github.com/Koenkk/zigbee2mqtt.io/blob/master/docs/devices/WV704R0A0902.md)*
+
+# Drayton Wiser Smart Radiator Thermostat (iTRV)
```
Comment (sjorge) [inline docs/devices/WV704R0A0902.md]: Should also be 'Schneider Electric'

File: docs/devices/WV704R0A0902.md [hunk 8]
```diff
@@ -0,0 +1,211 @@
+---
+title: "Drayton Wiser iTRV control via MQTT"
```
Comment (sjorge) [inline docs/devices/WV704R0A0902.md]: Should also be 'Schneider Electric'

File: docs/devices/WV704R0A0902.md (hunk 1 above)
Comment (arcanefoam) [reply docs/devices/WV704R0A0902.md]: Fixed

File: docs/devices/WV704R0A0902.md (hunk 3 above)
Comment (arcanefoam) [reply docs/devices/WV704R0A0902.md]: Fixed

File: docs/devices/WV704R0A0902.md (hunk 5 above)
Comment (arcanefoam) [reply docs/devices/WV704R0A0902.md]: Fixed

File: docs/devices/WV704R0A0902.md (hunk 7 above)
Comment (arcanefoam) [reply docs/devices/WV704R0A0902.md]: Fixed

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**